.gitignore

```diff
@@ -1,6 +1,29 @@
 # Please keep this list in ASCII sort order
+!.mvn/wrapper/maven-wrapper.jar
```

Comment:
They were added to the .gitignore automatically by [Spring Boot Initializr](https://start.spring.io/) and I decided to leave them there to avoid accidental commits

Reply:
I still don't understand their purpose.
Isn't this initializr script a thing to be run once to generate a backbone for the repo?

>I decided to leave them there to avoid accidental commits
I don't follow. Could you give an example of a situation with such accidental commits?

Reply:
I looked throw them once again and it turned out that you were right. Sorry :) Will remove them